packages/SwingSet/test/vo-test-harness/test-dvo-test-harness.js

```diff
@@ -0,0 +1,24 @@
+import { test, runDVOTest } from '../../tools/vo-test-harness.js';
+
+function bfile(name) {
+  return new URL(name, import.meta.url).pathname;
+}
+
+async function dvoTestTest(t, mode) {
+  function testLogCheck(_t, phase, log) {
+    t.deepEqual(log, ['start test', phase, 'test thing', { mode }, 'end test']);
+  }
+  await runDVOTest(t, testLogCheck, bfile('vat-dvo-test-test.js'), { mode });
+}
+
+test.failing('fail during "before" phase', async t => {
```

Comment:
I think you want a passing test that expects a throw.

Reply:
Not relevant in this case.